jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -110,6 +113,7 @@ public ImagesAndRegistryClient call()
     } else if (imageReference.getDigest().isPresent()) {
       Optional<Image> image = getCachedBaseImage();
       if (image.isPresent()) {
+        verifyPlatformOfCachedManifest(image.get());
```

Comment:
should this just trigger a redownload of the correct manifest?

Reply:
I mean we're in online mode, we know the manifest needs to be updated? Or is this just an interim solution to a problem that will be fixed?

Reply:
Oh! Why haven't I thought about it?

Reply:
If it's just short term. This is fine probably.

Reply:
It's a super easy fix and really nice to have. It's just that I really had hard time making the test work. Done.